core/src/main/python/update.py

```diff
@@ -144,15 +144,14 @@ def __update_online(model, model_context, aliases):
         __release_edit_session_and_disconnect()
         raise de
 
-    exit_code = 0
+    topology_updater.set_server_groups()
```

Comment:
maybe move this between update() and deployResources()

Reply:
This change does not seem related to the rest of the changes.  Why is it necessary to add the set_server_groups here?

Reply:
Never mind, I need to be in this code. I don't believe your code will work with extension templates by removing the edit, start edit. But my changes will remove the 3 statements.

Reply:
It's not really adding the logic set_server_groups now, it was there before and just move it earlier when it is still in edit session.